hyacinth/session.py

```diff
@@ -146,21 +146,25 @@ def get_matters(self, **kwargs):
         url = Session.__make_url("matters")
         return self.__get_paginated_resource(url, **kwargs)
 
+    def get_folder(self, id, **kwargs):
+        """GET a Document."""
+        url = Session.__make_url(f"folders/{id}")
+        return self.__get_resource(url, **kwargs)
+
+    def get_folders(self, **kwargs):
+        """GET a Document."""
+        url = Session.__make_url(f"folders")
+        return self.__get_resource(url, **kwargs)
```

Comment:
```suggestion
        return self.__get_paginated_resource(url, **kwargs)
```
I think you'll want to use `__get_paginated_resource` for list resources, otherwise you won't get the subsequent pages.